moped-database/migrations/1727450092173_rename-school-beacon-component/up.sql

```diff
@@ -0,0 +1 @@
+UPDATE moped_components SET component_name = 'Beacon', component_subtype = 'School Zone' WHERE component_id = 39
```

Comment:
what would be the alternative to updating by `component_id`?

Reply:
John proposed in that thread that Mike linked above to do something like `where component_name like '%refuge%'- ` so I guess in this case it would have to be `where component_subtype = School Zone Beacon`?